integration/test_collection_model.py

```diff
@@ -170,17 +172,17 @@ def test_tenants(client: weaviate.Client):
 def test_multi_searches(client: weaviate.Client):
     client.collection.delete("TestMultiSearches")
 
-    class SearchTest(BaseProperty):
+    class TestMultiSearches(BaseProperty):
         name: Optional[str] = None
 
     name = "TestMultiSearches"
```

Comment:
name can go away, delete accepts the model directly. Please also check other tests